srccomepamshop/model/Vehicle.java

```diff
@@ -0,0 +1,79 @@
+package com.epam.dao.task1.entity;
+
+import java.util.Objects;
+
+public abstract class Vehicle {
```

Comment:
try  to make another hierarchy, not use the same I used before

Reply:
i dont think he has time to change it now:)